x/ibc/07-tendermint/types/update.go

```diff
@@ -115,7 +115,8 @@ func checkValidity(
 	}
 
 	// assert header height is newer than consensus state
-	if header.GetHeight() <= consState.Height {
+	height := clienttypes.NewHeight(0, header.GetHeight())
+	if consState.Height.GT(height) {
```

Comment:
hmm this is changing semantics. `consState.Height.EQ(height)` is an error, but that would not be caught here

should be `!height.GT(consState.Height)`